component-classes/index.d.ts

```diff
@@ -1,510 +1,513 @@
 export namespace pageIndicator {
-    let wrapper: string;
-    let dot: string;
-    let inactive: string;
-    let active: string;
+    const wrapper: string;
```

Comment:
Have you run `pnpm install` after rebasing this branch? It looks like there's an old typescript version used to build these types 🤔

Reply:
Ah. Thanks. Sorry. 🙈😄